src/cli/rustup_mode.rs

```diff
@@ -319,6 +319,12 @@ pub fn cli() -> App<'static, 'static> {
                                 .help("Force an update, even if some components are missing")
                                 .long("force")
                                 .takes_value(false),
+                        )
+                        .arg(
+                            Arg::with_name("allow-downgrade")
+                                .help("Installs the most recent toolchain which supports all required components, even than this is older than the currently installed toolchain")
```

Comment:
This string is very long, perhaps:

```suggestion
                                .help("Allow rustup to downgrade the toolchain to satisfy your component choice")
```